### PR TITLE
cic track start error handling

### PR DESCRIPTION
**Pull request by lashd**
error handling for:
- authentication failure
- missing .netrc
- missing authentication credentials

### CODE REVIEW

File: bin/support/ruby/lib/commands/track/errors.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
+module Commands
+  module Track
+    class NetrcMissingError < Thor::Error
+      def initialize
+        super '.netrc file missing. Add a .netrc file to your home folder with your github credentials in it'
```
Comment (jimconner) [inline bin/support/ruby/lib/commands/track/errors.rb]: Might be good to give a pointer to an example .netrc file here. We could tell them to `man netrc` but it is not clear from the manpage exactly what the contents should be (and the manpage says it is used for FTP accounts which is no longer entirely correct). They will also need to get the permissions set correctly on the file after they've created it too, so perhaps we need to include some more information around `chmod 600 ~/.netrc` too.

Comment (jimconner) [reply bin/support/ruby/lib/commands/track/errors.rb]: Replying to myself here... I've been thinking that we need an introduction video covering how to set up virtualbox, ubuntu, github account, forking, netrc, chmod etc anyway.. so perhaps that error message is fine as it is as we might have already explained the required steps in more detail anyway.

Comment (lashd) [reply bin/support/ruby/lib/commands/track/errors.rb]: Either that or a link to some participant docs telling them what to do. For now I'll leave it is as it.

Re: The malformed-fork error. I'll add some handling now.

Comment (lashd) [reply bin/support/ruby/lib/commands/track/errors.rb]: I've done this now. You will need to re-pull the courseware image to see it.

**Pull request merged**